src/azul/indexer/aggregate.py

```diff
@@ -124,6 +124,8 @@ def accumulate(self, value) -> bool:
             else:
                 assert False
         else:
+            logger.warning('SetAccumulator reached max size of %d with value: %r',
+                           self.max_size, value)
```

Comment:
```suggestion
            logger.warning('Not aggregating value %r because accumulator is at capacity (%d).',
                           value, self.max_size)
```

Please look for other size-constrained accumulators and make sure that the same warning is logged there, too.

Reply:
Decided at PL not to add the warning in this PR due to https://github.com/DataBiosphere/azul/issues/4630